vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,150 @@
+4. Choose an empty namespace for `Installed Namespace`. You can create one from `Home → Projects` first, if needed, e.g. `entando-one`. 
+5. Click `Install` to install the operator into your target namespace
+6. The install may take a few minutes to complete after which you can click `View Operator` to see the operator in your namespace, or you can go to `Operators → Installed Operators` at any point and select it from there. 
+7. Now go to `EntandoApp` and click `Create EntandoApp`
+   - Keep the default `my-app` for your application name or select your own
+   - Select the EntandoApp version: `7.0`
+8. Provide an `Ingress Host Name` specific to your namespace, e.g. `my-app.YOUR-BASE-OPENSHIFT-URL.` In CRC you can keep the default `entando.apps-crc.testing` for your first project.
+9. Change the `DBMS` to `embedded`. This is the lightest and quickest way to test a full Entando Application but your preferred relational database is strongly recommended for production use.
+10. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources. 
+11. Go to `EntandoApp → my-app` to check the status of the deploy
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 2 - PostgreSQL
+ For this scenario we'll prepare a more production-like configuration. Here we'll switch from an embedded on-disk database to a dedicated PostgreSQL database. For this scenario we'll start where step 6 ends in the previous scenario. If you already ran Scenario 1 you can either go to that project and remove the EntandoApp and ProvidedCapabilities via `Installed Operators → Entando Operator` or prepare a new project using steps 1-5 above.
+ 
+Now let's create a new application, this time using PostgreSQL.
+1. Go to `EntandoApp` and click `Create instance`
+   - Keep the default `my-app` for your application name or select your own
+   - Select the EntandoApp version: `7.0`
+2. Set the `Ingress Host Name` as in Scenario 1 above
+3. Keep the default `DBMS` as `postgresql`  
+4. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources just as in Scenario 1 but with the addition of a PostgreSQL database deployment.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 3 - PostgreSQL plus OpenShift SSL
+For this scenario we'll setup PostgreSQL, just like Scenario 2, but also enable SSL using OpenShift's internal Certificate Authority (CA). As a starting point, you can either remove the EntandoApp and ProvidedCapabilities using the Operator or you can prepare a new project per steps 1-5 in Scenario 1.
+
+1. Using your browser, examine and save the SSL certificate for your environment. For example, in Chrome, go to any OpenShift console page, click on `View Site Information` next to the URL (it may be labeled `Not secure` ) then go to `Details -> Copy to File` and save the certificate as a Base-64 encoded X.509 file, e.g. `openshift.cer`.
```

Comment:
remove space after `Not secure`

comma after `Not secure`)

`Details -> Copy to File` -> `Details` -> `Copy to File` (and use the arrow symbol instead of a hyphen and carrot)